### ISSUE TITLE
The wiki/documentation sucks and isnt really helpful

### ISSUE THREAD

**Issue by XxImaDarkReaperxX** [Lauriethefish/QuestPatcher#141]
The wiki/documentation doesn't say what files in a qmod actually are. There just zip files yes but what do i put in the zip? How do i get the .so files that i see in the example qmod files? this is just confusing as there isn't any documentation for quest patcher at all besides the barely useful one here.

**Comment by qe201020335** [comment]
This is a tool to install mods to your quest game. It's not a tool to develop mods, mate. 

Here is a template for making qmods: https://github.com/Lauriethefish/quest-mod-template 
It is mainly used to develop mods for beat saber but it should be the same process for other supported games.

**Comment by Lauriethefish** [comment]
You're right, the documentation here for qmods isn't very good. While it does explain all parts of the qmod format, it doesn't clarify what the SO files need to be.
This is due to the general lack of documentation in quest modding - I would ideally be able to link to some QuestLoader/Scotland2 documentation explaining how a mod SO file needs to be formatted to be loaded by the modloader, but such a thing doesn't exist.

**Comment by RedBrumbler** [comment]
pretty sure at this point at least the readme for sl2 has some information about what a mod .so is expected to contain if you want anything to happen besides it being loaded
